### ISSUE TITLE
CRISCO new URL

### ISSUE THREAD

**Issue by arsaber101** [milouse/flycheck-grammalecte#24]
CRISCO has  a new url (without redirection...) :
https://crisco4.unicaen.fr/des/synonymes/synonyme

**Comment by milouse** [comment]
Thank you for the notification. I just fix it quickly. I wonder if it worth moving the CRISCO URL in a variable to allow one to change it?

**Comment by arsaber101** [comment]
Maybe. Or a warning in return that the connexion failed. For now it just returns `Aucun synonyme Aucun antonyme,`, which is not true.

It is not the first time CRISCO changes its URL. That's why I suspected a failure to connect. Before I came across your package (Thanks for it by the way) I used for years a couple of shell scripts I wrote (badly, I am not a programmer at all) from within emacs. They returned me a w3m-dump, roughly formatted throught pipe, of my requests. Because I encountered problems of failure with CRISCO, I changed the synonyme-script to point to [CNRTL Synonymie](https://www.cnrtl.fr/synonymie/) instead. Datas are the same (CRISCO is the provider of CNRTL), but CNRTL proves to be far more reliable. It could be another solution, but would implies a complete rewriting of the functions `grammalecte-find-synonyms` and `grammalecte-find-synonyms-at-point`, and the writing of two others functions for `Antonymes`, as they are not at the same URL as `Synonymes` in CNRTL site...

Thank again for your work, by far more elegant than what I was able to scribble.

While I am here, I open a Feature Request for a really cosmetic (but annoying) point&#x2026;
